src/rpc/governance.cpp

```diff
@@ -1005,7 +1005,9 @@ static UniValue voteraw(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Malformed base64 encoding");
     }
 
-    auto dmn = node.dmnman->GetListAtChainTip().GetValidMNByCollateral(outpoint);
+    CHECK_NONFATAL(node.dmnman);
```

Comment:
maybe better to implement `EnsureMNMan` same as `EnsureMempool` and `EnsureLLMQContext` ?

Reply:
Don't know if we should implement `Ensure`(`Any`)`*` unless it's significant (like `LLMQContext`)